castervoice/apps/sublime.py

```diff
@@ -10,6 +10,15 @@
 from castervoice.lib.dfplus.merge.mergerule import MergeRule
 from castervoice.lib.dfplus.state.short import R
 
+def action_lines(action, n, nn):
```

Comment:
Very clever. When Action Context becomes available. We can move this kind of function into AppStandard